sdk/servicebus/azure-servicebus/conftest.py

```diff
@@ -9,9 +9,24 @@
 import uuid
 
 import pytest
+from devtools_testutils import test_proxy
+from devtools_testutils.sanitizers import add_remove_header_sanitizer, add_general_regex_sanitizer, set_custom_default_matcher
+
 
 collect_ignore = []
 
+@pytest.fixture(scope="session", autouse=True)
+def add_aeg_sanitizer(test_proxy):
+    set_custom_default_matcher(
+        compare_bodies=False, excluded_headers="Authorization,Content-Length,x-ms-client-request-id,x-ms-request-id,ServiceBusSupplementaryAuthorization,ServiceBusDlqSupplementaryAuthorization"
```

Comment:
We can remove the first few from here. Aside from `Content-Length`, these headers are already [excluded from recordings](https://github.com/Azure/azure-sdk-for-python/blob/ff97aa8a61dc6accc72ee11441f4909c32d7695e/tools/azure-sdk-tools/devtools_testutils/proxy_startup.py#L185) with a `remove_header_sanitizer` when the proxy is started up -- those sanitizers are additive, so it doesn't get overridden by the call on line 23 here. `Content-Length` doesn't seem to affect execution for these tests (based on a local run of playback tests)
```suggestion
        compare_bodies=False, excluded_headers="ServiceBusSupplementaryAuthorization,ServiceBusDlqSupplementaryAuthorization"
```